templates/Dockerfile.j2

```diff
@@ -1,14 +1,20 @@
 # This Dockerfile was generated from templates/Dockerfile.j2
-{% set plugin_list = 'x-pack ingest-user-agent ingest-geoip' %}
-{% if staging_build_num -%}
-{%   set version_tag = elastic_version + '-' + staging_build_num -%}
-{%   set url_root = 'http://staging.elastic.co/%s/downloads/elasticsearch' % version_tag -%}
+{% set ingest_plugins = 'ingest-user-agent ingest-geoip' %}
+{% set version_tag = elastic_version -%}
+{% set tarball = 'elasticsearch-%s.tar.gz' % elastic_version -%}
+
+{% if release_manager and elastic_version.endswith('-SNAPSHOT')-%}
+{%   set url_root = 'http://localhost/build/snapshot-artifacts/elasticsearch/distribution/tar/build/distributions' -%}
+{%   set x_pack_url_root = 'http://localhost/build/snapshot-artifacts/x-pack-elasticsearch/plugin/build/distributions' -%}
+{%   set plugin_url_root = 'http://localhost/build/snapshot-artifacts/elasticsearch/plugins/$PLUGIN/build/distributions' -%}
+{% elif elastic_version.endswith('-SNAPSHOT') -%}
+{%   set url_root = 'https://snapshots.elastic.co/downloads/elasticsearch' -%}
```

Comment:
As discovered this will only work for published releases; for unreleased stuff the plugin installation will fail later on.
Maybe we can remove this section (`-SNAPSHOT` conditional) for now.